frontend/src/components/Apps/Apps.tsx

```diff
@@ -0,0 +1,38 @@
+import styles from './Apps.module.css';
+import { useState } from 'react';
+import App from './App/App';
+import AddAppModal from './AppModal/AddAppModal';
+
+const Apps: React.FunctionComponent = () => {
```

Comment:
It's awesome to type everything but should we type arrow functions as function components?

Reply:
Good point, new to typescript so I had no clue there were arrow function types just always assumed everything was FC typed. Will change.

Reply:
No worries, I think I was saying that defining React Components by class or by functions are pretty well-known so we might not have to type it : )